From 4e0369b4fcebde195a38d5b90c73b08f28e0047a Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Thu, 21 Jan 2021 11:15:02 +0100
Subject: [PATCH] kblrt5660: Fix file permissions

The exec bit was set wrongly on those files.  Fix them.

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 ucm2/kblrt5660/Hdmi1.conf     | 0
 ucm2/kblrt5660/Hdmi2.conf     | 0
 ucm2/kblrt5660/HiFi.conf      | 0
 ucm2/kblrt5660/kblrt5660.conf | 0
 4 files changed, 0 insertions(+), 0 deletions(-)
 mode change 100755 => 100644 ucm2/kblrt5660/Hdmi1.conf
 mode change 100755 => 100644 ucm2/kblrt5660/Hdmi2.conf
 mode change 100755 => 100644 ucm2/kblrt5660/HiFi.conf
 mode change 100755 => 100644 ucm2/kblrt5660/kblrt5660.conf

diff --git a/ucm2/kblrt5660/Hdmi1.conf b/ucm2/kblrt5660/Hdmi1.conf
old mode 100755
new mode 100644
diff --git a/ucm2/kblrt5660/Hdmi2.conf b/ucm2/kblrt5660/Hdmi2.conf
old mode 100755
new mode 100644
diff --git a/ucm2/kblrt5660/HiFi.conf b/ucm2/kblrt5660/HiFi.conf
old mode 100755
new mode 100644
diff --git a/ucm2/kblrt5660/kblrt5660.conf b/ucm2/kblrt5660/kblrt5660.conf
old mode 100755
new mode 100644
-- 
2.26.2

